docs/USAGE.md

```diff
@@ -0,0 +1,158 @@
+# Usage Guide
+
+> Klik disini untuk [dokumentasi penggunaan namun dalam Bahasa Indonesia](./USAGE_id.md)
+
+In this document you will find how to use Teknologi Umum Bot. Its' commands and available parameters.
+
+## Contents
+
+- 💙 [`/blidingej`](#-blidingej)
+- 💙 [`/quiz`](#-quiz)
+- 🌏📬 [`/covid`](#-covid)
+- 🌏📬 [`/devread`](#-devread)
+- 🌏 [`/dukun`](#-dukun)
+- 🌏📬 [`/eval`](#-eval)
+- 🌏📬 [`/search`](#-search)
+- 🌏📬 [`/snap`](#-snap)
+- 🌏📬 [`/help`](#-help)
+- 🛩📬 [`/kktbsys`](#-kktbys)
+- 🛩📬 [`/illuminati`](#-illuminati)
+- 🛩📬 [`/joke`](#-joke)
+- 🛩📬 [`/quote`](#-quote)
+- 🛩📬 [`/yntkts`](#-yntkts)
+
+Legends:
+
+- 💙 Teknologi Umum group only
+- 🌏 Available on groups
+- 🛩 Available on groups less than 500 members
+- 📬 Available on private chats
+- 🚧 Experimental feature
+- 🚨 Deprecated
+
+## `/blidingej`
+
+Will send you the definition of bliding ej.
+
+Available only on Teknologi Umum Group.
+
+- `/blidingej`
+- `/blidingej indo` or `/blidingej id` - Indonesian translation
+
+## `/quiz`
+
+Will send you a random quiz. Whether it'd be TOKI, general programming, or some tech-related trivia.
+
+Available only on Teknologi Umum Group.
+
+- `/quiz`
+
+## `/covid`
+
+Will send you Covid-19 statistic information
+
+- `/covid` - Global & Indonesia statistic
+- `/covid <country>` - That specific country statistic
+
+<img src="../.github/images/covid.jpg" width="300px" />
+
+## `/devread`
+
+Will send you a collection of developer articles and posts around a certain topic from pulo.dev.
+
+- `/devread <topic>`
+
+<img src="../.github/images/devread.jpg" width="300px" />
+
+## `/dukun`
+
+Dukun is an Indonesian term for shaman or sorcerer.
+It's kind of a joke for specifying those who could answer a given problem without any details about it whatsoever.
+
+Not available on private chats.
+
+Reply to other person's message with `/dukun +<point>` or `/dukun -<point>` to give them a dukun point.
+Maximum point given for a person is +10 and -10.
+
+Be careful of using this, it's a competitive life.
+
+- `/dukun` - Shows dukun leaderboard
+- `/dukun +<point>` - Increment someone's dukun point
+- `/dukun -<point>` - Decrement someone's dukun point
+
+<img src="../.github/images/dukun-leaderboard.jpg" width="300px" />
+<img src="../.github/images/dukun.jpg" width="300px" />
+
+## `/eval`
+
+Evaluate javascript expression, and shows the output. Bear in mind we limit the usage of a lots of stuff.
+If you want to produce more powerful things, please create it on [JSBin](https://jsbin.com/),[JSFiddle](http://jsfiddle.net/),
+[Codepen](https://codepen.io/), [Codesandbox](https://codesandbox.io/), or even [Repl.it](https://repl.it/).
+
+- `/eval <javascript expression>`
+
+<img src="../.github/images/eval.jpg" width="300px" />
+
+## `/search`
+
+Shows you 10 results from a DuckDuckGo search. As simple as that. No ads. Filtered clean search results by the algorithm.
```

Comment:
ackshhually, 10 + BEST (defaultnya 2, ini yg punya preview)

Reply:
well yeah but you wrote it as "10 result", so don't confuse people🤷🏻‍♂️